day_one/test.js

```diff
@@ -0,0 +1,15 @@
+import test from "ava";
+import { calculateFuelRequired, calculateTotalFuel } from "./index.js";
+
```

Comment:
smart!

Reply:
Thanks! It made sense seeing as they provided test cases for us :)